pages/users_and_roles.rst

```diff
@@ -16,7 +16,7 @@ The following sections describe the capabilities of users and roles **and also h
 
    users_and_roles/users
    users_and_roles/roles
-   users_and_roles/system_users
+   users_and_roles/permission_system
```

Comment:
Please add a placeholder page similar to https://github.com/Graylog2/documentation/blob/2.2/pages/load_balancers.rst so that people won't get a 404 when accessing the old URI.